packages/OmegaPrint-Core.package/OPPrinter.class/class/initialize.st

```diff
@@ -1,6 +1,7 @@
 class initialization
 initialize
 
-	CodeHolder
-		addContentsSymbolQuint: self omegaPrintQuint
-		afterEntry: #prettyPrint.
\ No newline at end of file
+	self isAlreadyInstalled
+		ifFalse: [ CodeHolder
+						addContentsSymbolQuint: self omegaPrintQuint
+						afterEntry: #prettyPrint ]
```

Comment:
das sieht in Squeak etwas anders aus und ist unter dem CodeHolder

Reply:
GitHub nimmt fuer die Anzeige andere Tabbreiten als Squeak das tut